pam/integration-tests/testdata/TestCLIAuthenticate/golden/authenticate_user_and_reset_password_while_enforcing_policy

```diff
@@ -32,7 +32,7 @@ Username: user name
 
 ────────────────────────────────────────────────────────────────────────────────
 > ./pam_authd login socket=${AUTHD_TESTS_CLI_AUTHENTICATE_TESTS_SOCK}
-Username: user-needs-reset
+Username: user-needs-reset-integration-mandatory
```

Comment:
Nope, because we use two different servers (authd instances) for those tests.

Ideally in future we could try to use just one server for most of the tests, unless a specific case is required (as we were doing in the past), but since it implies changing more users (and so golden files), as you rightfully pointed, I prefer to do that in another PR.

Here I did for some cases only since I wanted to be able to share the native tapes as much as possible with ssh ones.

Reply:
> Nope, because we use two different servers (authd instances) for those tests.

Ah, and the users of two authd test instances don't interfere with each other because they use different databases, right?

Reply:
Exactly see https://github.com/ubuntu/authd/blob/main/internal/testutils/daemon.go#L61-L132

Again I think we should try to stress-test the daemon by try to run as much as possible of the tests in the same instance though, so in another PR I may make that happen more.